panic: sql: expected 4 destination arguments in Scan, not 2

```
panic: sql: expected 4 destination arguments in Scan, not 2

goroutine 1 [running]:
main.panicIf(0x2b454f4868c0, 0xc2080d07f0)
        /home/vagrant/src/github.com/yuya-takeyama/db2yaml/main.go:291 +0x50
main.func·001(0xc2080740c0)
        /home/vagrant/src/github.com/yuya-takeyama/db2yaml/main.go:81 +0x12d
github.com/codegangsta/cli.(*App).Run(0xc208034340, 0xc20800a000, 0x9, 0x9, 0x0, 0x0)
        /home/vagrant/src/github.com/codegangsta/cli/app.go:161 +0xd5a
main.main()
        /home/vagrant/src/github.com/yuya-takeyama/db2yaml/main.go:85 +0x533

goroutine 5 [chan receive]:
database/sql.(*DB).connectionOpener(0xc20806caa0)
        /usr/local/go/src/database/sql/sql.go:589 +0x4c
created by database/sql.Open
        /usr/local/go/src/database/sql/sql.go:452 +0x31c

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
        /usr/local/go/src/runtime/asm_amd64.s:2232 +0x1
```

Changed error messages in 7c00b6ee7b142fe52fffea8e9efc30d44ca9e2c1 and got below.

```
panic: failed to load data structure: failed to load tables: failed to read DDL: sql: expected 4 destination arguments in Scan, not 2

goroutine 1 [running]:
main.panicIf(0x2b99b17268c0, 0xc2081367a0)
        /home/vagrant/src/github.com/yuya-takeyama/db2yaml/main.go:290 +0x50
main.func·001(0xc2080740c0)
        /home/vagrant/src/github.com/yuya-takeyama/db2yaml/main.go:81 +0x12d
github.com/codegangsta/cli.(*App).Run(0xc208034340, 0xc20800a000, 0x9, 0x9, 0x0, 0x0)
        /home/vagrant/src/github.com/codegangsta/cli/app.go:161 +0xd5a
main.main()
        /home/vagrant/src/github.com/yuya-takeyama/db2yaml/main.go:85 +0x533

goroutine 5 [chan receive]:
database/sql.(*DB).connectionOpener(0xc20806a6e0)
        /usr/local/go/src/database/sql/sql.go:589 +0x4c
created by database/sql.Open
        /usr/local/go/src/database/sql/sql.go:452 +0x31c

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
```

So the origin of this problem seems here.
https://github.com/yuya-takeyama/db2yaml/blob/d94a2f1d140348ac08785fa95f84c13a46f719a6/main.go#L152

--- a/main.go
+++ b/main.go
@@ -123,7 +123,7 @@ func loadDatabaseStructure(conn *sql.DB, databaseName string) (*model.Database, 
 }
 
 func loadTables(conn *sql.DB, databaseName string, database *model.Database) error {
-	stmt, err := conn.Prepare("SELECT `TABLE_NAME`, `TABLE_COMMENT` FROM `information_schema`.`TABLES` WHERE `TABLE_SCHEMA` = ?")
+	stmt, err := conn.Prepare("SELECT `TABLES`.`TABLE_NAME`, `TABLES`.`TABLE_COMMENT` FROM `information_schema`.`TABLES` LEFT JOIN `information_schema`.`VIEWS` ON `TABLES`.`TABLE_SCHEMA` = `VIEWS`.`TABLE_SCHEMA` AND `TABLES`.`TABLE_NAME` = `VIEWS`.`TABLE_NAME` WHERE `TABLES`.`TABLE_SCHEMA` = ? AND `VIEWS`.`TABLE_NAME` IS NULL")
 
 	if err != nil {
 		return fmt.Errorf("failed to prepare statement to read table informations: %s", err)
@@ -220,7 +220,9 @@ func loadColumns(conn *sql.DB, databaseName string, database *model.Database) er
 
 		table := database.Tables[tableName]
 
-		table.AddColumn(column)
+		if table != nil {
+			table.AddColumn(column)
+		}
 	}
 
 	return nil

